vdj_ann/src/annotate.rs

```diff
@@ -3018,6 +3018,8 @@ pub struct ContigAnnotation {
     pub is_asm_cell: Option<bool>, // Was the barcode declared a cell by the VDJ assembler
 
     pub full_length: Option<bool>, // New field added in CR 4.1. None if the field is not set
+
+    pub junction_support: Option<(i32, i32)>, // New field added in CR 7.2. Number of (umi,reads) covering junction region for good contig
```

Comment:
Would be better to define a struct for `JunctionSupport`
```
pub struct JunctionSupport {
    reads: u32,
    umis: u32,
}
```
```suggestion
    pub junction_support: Option<JunctionSupport>, 
```

Reply:
Done.